test/hello_world.test.js

```diff
@@ -1,120 +0,0 @@
-var test = require('tape');
```

Comment:
@springmeyer Shall I plan to re-add this to a separate branch, like `class-example`?